src/components/BookInfo.css

```diff
@@ -0,0 +1,35 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@200;400;600&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@200;400;600&display=swap');
+
+.Info-Box {
+  flex: 3;
+}
+
+.Cat-Info {
+  display: block;
```

Comment:
**Changes Required**
I must say your CSS files are well organized and modular, bravo. However;
- [ ] The CSS selector property `display` with the value of `block` is repeated multiple times. You want to follow best CSS practice of Don't Repeat Yourself. (DRY). Please let us refactor it by creating for instance a `.display-block` in your `.jsx` file and having a `.display-block { display: block}` in your CSS file. Please do this for all repeating Selector properties in all your CSS files.